dashboard/modules/reporter/reporter_head.py

```diff
@@ -24,6 +24,7 @@ class ReportHead(dashboard_utils.DashboardHeadModule):
     def __init__(self, dashboard_head):
         super().__init__(dashboard_head)
         self._stubs = {}
+        self._ray_config = None
```

Comment:
intentional?

Reply:
Yep, this is a fix for a bug that I added during iteration on a PR from a previous PR.